pkg/install/genevalogging.go

```diff
@@ -0,0 +1,399 @@
+				ObjectMeta: metav1.ObjectMeta{
+					Annotations: map[string]string{"scheduler.alpha.kubernetes.io/critical-pod": ""},
+					Labels:      map[string]string{"app": "mdsd"},
+				},
+				Spec: v1.PodSpec{
+					Volumes: []v1.Volume{
+						{
+							Name: "hostlog",
+							VolumeSource: v1.VolumeSource{
+								HostPath: &v1.HostPathVolumeSource{
+									Path: "/var/log",
+								},
+							},
+						},
+						{
+							Name: "mdsd-config",
+							VolumeSource: v1.VolumeSource{
+								ConfigMap: &v1.ConfigMapVolumeSource{
+									LocalObjectReference: v1.LocalObjectReference{Name: "mdsd-config"},
+								},
+							},
+						},
+						{
+							Name: "fluentd-config",
+							VolumeSource: v1.VolumeSource{
+								ConfigMap: &v1.ConfigMapVolumeSource{
+									LocalObjectReference: v1.LocalObjectReference{Name: "fluentd-config"},
+								},
+							},
+						},
+						{
+							Name: "gcs-cert",
+							VolumeSource: v1.VolumeSource{
+								Secret: &v1.SecretVolumeSource{
+									SecretName: "gcs-cert",
+								},
+							},
+						},
+						{
+							Name: "socket",
+							VolumeSource: v1.VolumeSource{
+								EmptyDir: &v1.EmptyDirVolumeSource{},
+							},
+						},
+					},
+					HostPID: true,
+					ImagePullSecrets: []v1.LocalObjectReference{
+						{
+							Name: "azure-registry",
+						},
+					},
+					ServiceAccountName: "geneva",
+					Tolerations: []v1.Toleration{
+						{
+							Effect:   v1.TaintEffectNoExecute,
+							Operator: v1.TolerationOpExists,
+						},
+						{
+							Effect:   v1.TaintEffectNoSchedule,
+							Operator: v1.TolerationOpExists,
+						},
+					},
+					Containers: []v1.Container{
+						{
+							Name:  "td-agent",
+							Image: "TODO",
+							Env: []v1.EnvVar{
+								{
+									Name:  "FLUENTD_CONF",
+									Value: "config/fluent.conf",
+								},
+							},
+							Resources: v1.ResourceRequirements{
+								Limits: v1.ResourceList{
+									"cpu":    resource.MustParse("500m"),
+									"memory": resource.MustParse("200Mi"),
+								},
+								Requests: v1.ResourceList{
+									"cpu":    resource.MustParse("500m"),
+									"memory": resource.MustParse("200Mi"),
+								},
+							},
+							SecurityContext: &v1.SecurityContext{
+								Privileged: to.BoolPtr(true),
+								RunAsUser:  to.Int64Ptr(0),
+							},
+							VolumeMounts: []v1.VolumeMount{
+								{
+									Name:      "socket",
+									ReadOnly:  true,
+									MountPath: "/var/run/mdsd/",
+								},
+								{
+									Name:      "fluentd-config",
+									MountPath: "/td-agent/config",
+								},
+								{
+									Name:      "hostlog",
+									MountPath: "/var/log/",
+								},
+							},
+						},
+						{
+							Name:    "mdsd",
+							Command: []string{"mdsd"},
+							Args:    []string{"-D", "-R", "-c", "/etc/mdsd.d/config/mdsd.xml"},
+							Image:   "", //todo
+							Env: []v1.EnvVar{
+								{
+									Name:  "MONITORING_GCS_ACCOUNT",
+									Value: "", //todo
```

Comment:
again, pipe these through from rp-production.json - env vars make sense for these